scanapi/reporter.py

```diff
@@ -33,3 +34,27 @@ def write(self, responses):
 
         logger.info("The documentation was generated successfully.")
         logger.info(f"It is available at {self.output_path}")
+
+    def hide_headers_info(self, responses):
+        hide_keys = self.hide_keys()
+
+        if not hide_keys:
+            return
+
+        [
+            self.hide_request_headers_info(response.request, hide_keys)
+            for response in responses
+        ]
+
+    def hide_request_headers_info(self, request, hide_keys):
+        request_headers = request.headers
+
+        for key in hide_keys:
+            if key in request_headers:
+                request_headers[key] = "<sensitive_information>"
+
+    def hide_keys(self):
+        if not SETTINGS.get("report") or not SETTINGS["report"]["hide"]:
```

Comment:
Shouldn't we use `.get("hide")` in the second statement to avoid a `KeyError: 'hide'` as well?

Reply:
The config file would be something like this:

```
report:
  hide:
    headers:
      - Authorization
      - Token
```

So first we need to check if we get `report` and `hide` keys. But I have to change  ````SETTINGS["report"]["hide"]```` to ```SETTINGS.get("hide")``` indeed.

Reply:
@barbosa I fix it! Good catch 👀 Thank you!! 🎉